docs/extensions/getting-started.md

```diff
@@ -13,9 +13,9 @@ Use the [custom blocks feature](/defining-blocks) for this purpose.
 
 You will need to get a [GitHub](https://github.com) account and create a GitHub repository. At this time, other repository hosting solutions (GitLab, Bitbucket, etc.) are not supported.
 
-From the main editor screen, go to the gear wheel and **Extensions**. Click on **GitHub login**
-at the bottom of the list
-and follow prompts to generate a personal access token (make sure to select the **repo** checkbox). 
+Create a new project from the home screen. In the editor, click on the **GitHub** button next to the
```

Comment:
Instead of asking to click the **Github** button next to the save button, can we ask user to login from the home screen **import** button?